slackbot/bot.py

```diff
@@ -39,6 +46,16 @@ def _keepactive(self):
             time.sleep(30 * 60)
             self._client.ping()
 
+    def _logging_setting(self):
+        kw = {
+            'format': '[%(asctime)s] %(message)s',
+            'datefmt': '%m/%d/%Y %H:%M:%S',
+            'level': logging.DEBUG if self._debug else logging.INFO,
+            'stream': sys.stdout,
+        }
+        logging.basicConfig(**kw)
+        logging.getLogger('requests.packages.urllib3.connectionpool').setLevel(logging.WARNING)
+
```

Comment:
`Slackbot` is a library, which means we should not do any logging settings in the code. Logging should be setup by the application that makes use of slackbot.